src/index.tsx

```diff
@@ -7,9 +7,11 @@ import React, { StrictMode } from 'react'
 import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
 import { HashRouter as Router } from 'react-router-dom'
-import { AnalyticsProvider } from '@gooddollar/web3sdk-v2'
+import { AnalyticsProvider, GoodIdContextProvider } from '@gooddollar/web3sdk-v2'
 import { NewsFeedProvider } from '@gooddollar/web3sdk-v2'
 import { PostHogProvider } from 'posthog-react-native'
+import { Provider as PaperProvider } from 'react-native-paper'
+import { GoodIdProvider, GoodUIi18nProvider, NativeBaseProvider, RedirectNoticeProvider } from '@gooddollar/good-design'
```

Comment:
can we consolidate this into a single provider?

Reply:
@sirpy do you mean as export from the package itself?
or just a separate import into a providers.tsx on protocolUI side?

Reply:
done here: https://github.com/GoodDollar/GoodProtocolUI/pull/578/commits/8a9f5215d88322cb669869aef878d721b31fc518